.python-version

```diff
@@ -1,4 +1,4 @@
-3.6.13
+
 3.7.10
 3.8.10
 3.9.5
```

Comment:
do you not need to add a 3.10 here? I'm reviewing each commit so apologies if this is addressed in a later one

Reply:
This should be addressed in a later commit. These are not strictly necessary at all but do help point to specific patch versions for local development.